packages/api/src/api/expoJudgingVote/post.ts

```diff
@@ -0,0 +1,46 @@
+import { Schema } from '@hangar/shared';
+import { Request, Response } from 'express';
+import { ExpoJudgingSession, ExpoJudgingVote, Judge } from '@hangar/database';
+import { validatePayload } from '../../utils/validatePayload';
+import { logger } from '../../utils/logger';
+
+export const post = async (req: Request, res: Response) => {
+  const { entityManager, judge } = req;
+  
+  const { errorHandled, data } = validatePayload({
+    req,
+    res,
+    schema: Schema.expoJudgingVote.post,
+  });
+  if (errorHandled) return;
+
+  const { currentProjectChosen, expoJudgingSessionId } = data
+
+  let expoJudgingSession: ExpoJudgingSession | undefined;
+  try{
+    expoJudgingSession = await entityManager.findOne(ExpoJudgingSession, {id:expoJudgingSessionId}) ?? undefined
+  } catch (error) {
+    logger.error(error);
+    res.sendStatus(500);
+    return;
+  }
```

Comment:
I'd prefer we get rid of this try catch block in favor of an all-encompassing one wrapped around this and the judge creation; you already have the 500 handler at the bottom so it'd be great to avoid the redundancy especially since neither has logic to handle different errors differently